src/sentry/static/sentry/app/views/settings/projectAlerts/alertTypeChooser.tsx

```diff
@@ -28,57 +26,58 @@ const IssuesTooltip = ({children}: {children?: React.ReactNode}) => (
 
 const AlertTypeChooser = ({selected, onChange}: Props) => (
   <Container>
-    <TypeCard interactive onClick={() => onChange('metric')}>
+    <TypeCard interactive onClick={() => onChange('issue')}>
       <RadioLabel>
         <Radio
-          aria-label="metric"
-          checked={selected === 'metric'}
-          onChange={() => onChange('metric')}
+          aria-label="issue"
+          checked={selected === 'issue'}
+          onChange={() => onChange('issue')}
         />
-        {t('Metric Alert')}
-        <FeatureBadge type="beta" />
+        {t('Issue Alert')}
```

Comment:
Did we want to kill the badge?

Reply:
That's what I understood from the notion outline